decode.rb

```diff
@@ -0,0 +1,29 @@
+def decode_char(char)
+  morse_alphabet = {
+    '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D', '.' => 'E',
+    '..-.' => 'F', '--.' => 'G', '....' => 'H', '..' => 'I', '.---' => 'J',
+    '-.-' => 'K', '.-..' => 'L', '--' => 'M', '-.' => 'N', '---' => 'O',
+    '.--.' => 'P', '--.-' => 'Q', '.-.' => 'R', '...' => 'S', '-' => 'T',
+    '..-' => 'U', '...-' => 'V', '.--' => 'W', '-..-' => 'X', '-.--' => 'Y',
+    '--..' => 'Z'
+  }
+  morse_alphabet[char]
+end
+
+def decode_morise_word(word)
+  chars = word.split
+  ele = ''
+
+  chars.each { |char| ele += decode_char(char) }
+
+  ele
+end
+
+def decode_morise_message(message)
+  ele = ''
+  message = message.split('   ')
+  message.each { |x| ele += "#{decode_morise_word(x)} " }
+  ele
+end
+
+decode_message('.-   -... --- -..-   ..-. ..- .-.. .-..   --- ..-.   .-. ..- -... .. . ...')
```

Comment:
- [ _Optional_ ] You don't have a method called `decode_message` kindly change it to `decode_morise_message` which is working perfectly.